### PR TITLE
VACMS-4614: Require image field on system nodes.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4614 

## Testing done
Visual / behat

## QA steps

As user _uid_ with _user_role_
As an administrator, go to `/node/add/health_care_region_page` and visually verify that field_media is required.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by swirtSJW** [comment]
This had a test failure that I think was a mistake, so I just triggered them again.

**Comment by swirtSJW** [comment]
@ethanteague Looks like the test failure was legit.  Can you look into it?

**Comment by ethanteague** [comment]
> @ethanteague Looks like the test failure was legit. Can you look into it?

Updated, @swirtSJW

File: composer.json [hunk 1]
```diff
@@ -289,7 +289,8 @@
                 "3188331 - Vertical tabs scroll behavior is buggy": "https://www.drupal.org/files/issues/2020-12-14/vertical-tab-scroll-frozen-2.patch",
                 "2893933 - claimItem in the database and memory queues does not use expire correctly": "https://www.drupal.org/files/issues/2020-01-07/2893933-34.patch",
                 "3198608 - Migrate does not set set track_lst_updated from config": "https://www.drupal.org/files/issues/2021-02-15/3198608-track-last-updated-6.patch",
-                "3176975 - Allow updating easyrdf/easyrdf on 8.9.x": "patches/allow_updating_easyrdf_8_9_x.patch"
+                "3176975 - Allow updating easyrdf/easyrdf on 8.9.x": "patches/allow_updating_easyrdf_8_9_x.patch",
+                "3205691 - Disallow aria-required on fieldsets": "https://www.drupal.org/files/issues/2021-03-25/disallow-aria-required-fieldsets.patch"
```
Comment (swirtSJW) [inline composer.json:293]: Hmmm.  This solution bothers me as it takes away this check... which is actually an important one.

Lets zoom on this one.

Comment (swirtSJW) [reply composer.json:293]: Nevermind... disrgard, ignore I said something stupid.

**Review by swirtSJW — APPROVED** [review]

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**